modules/client/src/controllers/ConditionalTransferController.ts

```diff
@@ -90,7 +90,7 @@ export class ConditionalTransferController extends AbstractController {
       recipientPublicKey.slice(2), // remove 0x
       preImage,
     );
-    const encryptedPreImage = EthCrypto.cipher.stringify(encryptedPreImageCipher);
+    const encryptedPreImage = EthCrypto.stringify(encryptedPreImageCipher);
     await this.connext.setRecipientAndEncryptedPreImageForLinkedTransfer(
       recipient,
       encryptedPreImage,
```

Comment:
The `encryptedPreImage` var isn't ever used here, it should be hidden w/in the crypto module